go/1_hello-world.go

```diff
@@ -19,10 +23,30 @@ func getSecret() (string, error) {
 		return "", fmt.Errorf("unable to initialize Vault client: %w", err)
 	}
 
-	// WARNING: Storing any long-lived token with secret access in an environment variable poses a security risk.
-	// Additionally, root tokens should never be used in production or against Vault installations containing real secrets.
+	// WARNING: Storing any long-lived token with secret access in an
+	// environment variable poses a security risk. Additionally, root tokens
+	// should never be used in production or against Vault installations
+	// containing real secrets.
+	//
 	// See the files starting in auth-* for examples of how to securely log in to Vault using various auth methods.
-	client.SetToken(os.Getenv("TOKEN")) // If this line is omitted, Vault will use whatever the env var VAULT_TOKEN is set to.
+
```

Comment:
Did we change to user pass here b/c the `SetToken` happens behind the scenes now? I'm curious how the token auth works now and if there is a way to change the env var that gets picked up by the Go SDK

Just curious if I should add user pass auth to dotnet, but I feel like it was more intended for cli use

Reply:
As we discussed, removed the insecure hello-world example altogether since we have enough "how to do this the wrong way" in the docs and not enough "how to do this the right way". 😄